FlaskApi/src/menu.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask
+from flask_restful import Resource, Api
+
+app = Flask(__name__)
+api = Api(app, prefix='/api/v1')
```

Comment:
Do not initialize more than one application. As mentioned above, initialize it once and import all resources to it.